shell.go

```diff
@@ -20,11 +22,42 @@ import (
 	tar "github.com/whyrusleeping/tar-utils"
 )
 
+const (
+	DefaultPathName = ".ipfs"
+	DefaultApiFile  = "api"
```

Comment:
Those were taken from https://github.com/ipfs/go-ipfs/blob/master/repo/config/config.go#L34, decided to leave for consistency